homeassistant/components/mobile_app/websocket_api.py

```diff
@@ -0,0 +1,85 @@
+"""Websocket API for mobile_app."""
+from aiohttp.http_websocket import WSMessage
+
+from homeassistant.components.cloud import async_delete_cloudhook
+from homeassistant.components.websocket_api import (ActiveConnection,
+                                                    async_register_command,
+                                                    async_response,
+                                                    error_message,
+                                                    result_message,
+                                                    ws_require_user)
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import (ATTR_DELETED_IDS, ATTR_REGISTRATIONS, ATTR_STORE,
+                    CONF_USER_ID, CONF_WEBHOOK_ID, DOMAIN,
+                    SCHEMA_WS_DELETE_REGISTRATION, SCHEMA_WS_GET_REGISTRATION,
+                    WS_TYPE_DELETE_REGISTRATION, WS_TYPE_GET_REGISTRATION)
+
+from .helpers import safe_device, savable_state
+
+
+def register_websocket_handlers(hass: HomeAssistantType) -> bool:
+    """Register the websocket handlers."""
+    async_register_command(hass, WS_TYPE_GET_REGISTRATION,
```

Comment:
Use new way of registering command https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/system_health/__init__.py#L56-L58